changes-set.txt

```diff
@@ -43,13 +43,13 @@ proposed  syl5ibcom imbitridcom
 proposed  syl5ibr   imbitrrid
 proposed  syl5ibrcom imbitrridcom
 proposed  syl5bb    bitrid      compare to bitri or bitrd
-proposed  syl5rbb   bitridcom
+proposed  syl5rbb   bitr2id
 proposed  syl5bbr   bitr3id     compare to bitr3i or bitr3d
-proposed  syl5rbbr  bitr3idcom
+proposed  syl5rbbr  bitr3di
```

Comment:
And would you change the order of the hypotheses? If so we could add "change order of hypotheses from syl5rbbr" to the notes column.